src/components/organisms/ChatDisplay.tsx

```diff
@@ -8,6 +8,7 @@ export interface ChatDisplayProps {
 }
 
 const ChatDisplay: FC<ChatDisplayProps> = ({ chat }) => {
+  const cleanChat = chat.filter((message: Message) => !message.isSync)
```

Comment:
This can have performance issues because it recreates the array. Why don't you just not `return null` in the component if it is sync msg?

Reply:
fair